scripts/build/package.js

```diff
@@ -91,10 +93,17 @@ data.icons.forEach(icon => {
   const jsFilepath = path.resolve(iconsDir, `${filename}.js`);
   minifyAndWrite(jsFilepath, `module.exports=${iconObject};`);
 
+  const dtsFilepath = path.resolve(iconsDir, `${filename}.d.ts`);
+  fs.writeFileSync(dtsFilepath, `
+  import { SimpleIcon } from "../types";
+  declare const icon: SimpleIcon
+  export default icon;`);
+
```

Comment:
It is necessary to produce an declaration file for each icon, I could not get TS to work with the `declare module` glob in `types/index.d.ts`. The package size increased by 7 MB (30 to 37). I see this a minor problem as we plan to remove support for the `icons/*` files anyway.